plugins/chat/spec/services/chat/trash_message_spec.rb

```diff
@@ -62,8 +62,11 @@
           freeze_time
           messages = nil
           event =
-            DiscourseEvent.track_events { messages = MessageBus.track_publish { result } }.first
-          expect(event[:event_name]).to eq(:chat_message_trashed)
+            DiscourseEvent
```

Comment:
Same thing, it was failing on CI: https://github.com/discourse/discourse/actions/runs/6166188952/job/16735266241?pr=23519